airflow/www/views.py

```diff
@@ -30,12 +30,13 @@
 from functools import wraps
 from json import JSONDecodeError
 from operator import itemgetter
-from typing import Any, Callable, List, Optional, Set, Tuple, Union
+from typing import Any, Callable, Dict, List, Optional, Set, Tuple, Union
 from urllib.parse import parse_qsl, unquote, urlencode, urlparse
 
 import lazy_object_proxy
 import markupsafe
 import nvd3
+import sqlalchemy
 import sqlalchemy as sqla
```

Comment:
There’s already an alias imported so you can use `sqla` for where you’re using `sqlalchemy` and onmit the import addition.